cice.setup

```diff
@@ -934,9 +936,21 @@ EOF
     if (-e ${fimods}) rm ${fimods}
     if (-e ${fsmods}) rm ${fsmods}
 
+    # Use an existing ice_in file from the suite if it exists
+    # to reduce time spend in parse_namelist
+    set skip_parse_namelist = spval
+    if (${dosuite} == 1) then
+      set iceinfn = ../ice_in_save_${grid}${soptions}
```

Comment:
I think I'd maybe prefer we put the saved namelists in a separate directory, so  as to not clutter the suite's top level directory, but I don't have that strong of an opinion about it.

Reply:
These sit in the test suite directory and are deleted at the end of the test suite.  You're right that it may be cleaner to put them in a separate directory, but I think we're OK for now.